src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, String color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        setColor(color);
+        obtainTheArea();
+    }
+
+    public void obtainTheArea() {
```

Comment:
The method name `obtainTheArea` is not very descriptive. Consider renaming it to something more indicative of its action, like `calculateArea`, and this method should not be public since it's an internal operation of the class.